pkg/controllers/namespace/subnet_handler.go

```diff
@@ -0,0 +1,336 @@
+package namespace
```

Comment:
It is not the same with other handlers. I think we need to rename the file to subnet_util.

Reply:
refactored as subnet_share_controller